### PR TITLE
[speedtest] migrate to common

### PR DESCRIPTION
**Pull request by nicholaswilde**
Signed-off-by: Nicholas Wilde <ncwilde43@gmail.com>

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/stable/speedtest/Chart.yaml [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+    email: jeff@billimek.com
 dependencies:
-- name: influxdb
-  version: 1.1.9
-  repository: https://charts.bitnami.com/bitnami
-  condition: influxdb.enabled
+  - name: influxdb
+    version: 2.2.1
+    repository: https://charts.bitnami.com/bitnami
+    condition: influxdb.enabled
```
Comment (bjw-s) [inline charts/stable/speedtest/Chart.yaml:22]: Does this app work well with influx v2? The values in `values.yaml` seem to indicate this was meant for Influx v1.x, as v2 has removed the concept of databases according to the [bitnami upgrade guide](https://github.com/bitnami/charts/tree/master/bitnami/influxdb#to-200)

**Comment by onedr0p** [comment]
@billimek are you still using this chart? Do you think it still has a purpose over speedtest-exporter?

**Comment by billimek** [comment]
> @billimek are you still using this chart? Do you think it still has a purpose over speedtest-exporter? We can deprecate it and point people over to the other chart.

I don't personally run this one anymore.  The only purpose I see over `speedtest-exporter` would be if someone chooses to use influxdb vs prometheus.

**Comment by onedr0p** [comment]
I'd suggest we deprecate it or rename it to speedtest-influxdb

**Comment by bjw-s** [comment]
I'd be fine with deprecating as well. We can leave the deprecated version in the repo for some time, and if nobody wants it back we can remove it altogether

**Comment by onedr0p** [comment]
@nicholaswilde I'll go ahead and create a PR to deprecate this.

**Comment by nicholaswilde** [comment]
Thanks. I am interested in how you guys do depreciation.

**Comment by onedr0p** [comment]
This is pretty much it

https://github.com/k8s-at-home/charts/pull/826

**Pull request closed without merging**